core/src/main/resources/lib/layout/pageHeader.jelly

```diff
@@ -0,0 +1,137 @@
+<!--
+The MIT License
+
+Copyright (c) 2004-2010, Sun Microsystems, Inc., Kohsuke Kawaguchi,
```

Comment:
I don't think the copywriter should be added to this new file.

Reply:
Removed!